bin/test-local.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
I don't like the name of this script. "test" implies running some tests to me. Maybe `bin/boot-cloud-from-local.sh`?

Reply:
yes, very good point